### PR TITLE
Dorman Hack Club Added Alphabetically

### PR DESCRIPTION
**Pull request by SerenityUX**

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -659,6 +659,10 @@ dinos:
 docs.haas:
   type: A
   value: 167.99.113.134
+dorman:
+ - ttl : 1
```
Comment (maggie-j-liu) [inline hackclub.com.yaml:663]:
```suggestion
   ttl: 1
```
should fix it

File: hackclub.com.yaml [hunk 2]
```diff
@@ -659,6 +659,10 @@ dinos:
...
+  type: ALIAS
+  value: cname.vercel-dns.com
```
Comment (sampoder) [inline hackclub.com.yaml:665]:
```suggestion
  value: cname.vercel-dns.com.
```

**Review by sampoder — CHANGES_REQUESTED** [review]
Small typo, probably my bad! Sorry!

File: hackclub.com.yaml [hunk 3]
```diff
@@ -659,6 +659,10 @@ dinos:
 docs.haas:
   type: A
   value: 167.99.113.134
+dorman:
+   ttl: 1
```
Comment (maggie-j-liu) [inline hackclub.com.yaml:663]:
```suggestion
  ttl: 1
```
oops

**Pull request merged**